lib/resque_bus/server.rb

```diff
@@ -22,6 +22,123 @@ def self.included(base)
         
       }
     end
+
+    class Helpers
+      class << self
+        def parse_query(query_string)
+          has_open_brace = query_string.include?("{")
+          has_close_brace = query_string.include?("}")
+          has_multiple_lines = query_string.include?("\n")
+          has_colon = query_string.include?(":")
+          has_comma = query_string.include?(",")
+          has_quote = query_string.include?("\"")
+
+          exception = nil
+
+          # first let's see if it parses
+          begin
+            query_attributes = JSON.parse(query_string)
+            raise "Not a JSON Object" unless query_attributes.is_a?(Hash)
```

Comment:
I think we could also just set the exception variable here. That would avoid the raise and immediate rescue.